foundryup/foundryup

```diff
@@ -98,16 +100,14 @@ main() {
 
     say "installing foundry (version ${FOUNDRYUP_VERSION}, tag ${FOUNDRYUP_TAG})"
 
-    PLATFORM="$(uname -s)"
+    PLATFORM=$(tolower "${FOUNDRYUP_PLATFORM:-$(uname -s)}")
     EXT="tar.gz"
     case $PLATFORM in
-      Linux)
-        PLATFORM="linux"
-        ;;
-      Darwin)
+      linux) ;;
+      darwin|mac*)
         PLATFORM="darwin"
         ;;
-      MINGW*)
+      mingw*|win*)
```

Comment:
was this tested? is this correct?

Reply:
Yes I've tested it with lower, upper, mixed case strings starts with mingw and win, such as MINGW32_NT-, win32, and Windows.